toolchain/sem_ir/formatter.cpp

```diff
@@ -139,9 +139,16 @@ class InstNamer {
       return BuiltinKind::FromInt(inst_id.index).label().str();
     }
 
+    if (inst_id == InstId::PackageNamespace) {
+      return "package";
+    }
+
     auto& [inst_scope, inst_name] = insts[inst_id.index];
     if (!inst_name) {
       // This should not happen in valid IR.
+      // TODO: This is encountered for names in namespaces: the namespace is
+      // declared before the name, and then the formatter prints the name scope
+      // for the name before it's been added.
```

Comment:
(Don't forget to remove this TODO now #3447 has landed.)

Reply:
Done, thanks for the reminder.